docs/reference/commands/node-tool.asciidoc

```diff
@@ -3,81 +3,90 @@
-will be imported as dangling indices into cluster B, and these shards will be
-treated as in-sync even if they were not in-sync in cluster A,
-which may confusingly expose an arbitrarily stale shard to searches.
-
-[NOTE]
-When we're talking about the loss of nodes, we mean loss of storage for these
-nodes. If you still have access to the nodes storage, you should copy
-previous nodes data to other nodes and start Elasticsearch.
-
-Starting with Elasticsearch 7 cluster does not really support these operations:
-
-* A cluster that loses a majority of its master-eligible nodes will not
-proceed until it finds a majority of precisely those nodes again, using the
-persistent node ID for identification. New nodes will have new persistent node
-IDs so will not count towards a majority.
-
-* When node tries to join different cluster, cluster UUID comparison will fail.
-
-There is value in supporting these operations for severe disasters,
-but this is never done automatically and requires manual user intervention and
-acceptance of the risk of data loss.
-
-[float]
-=== Strongly prefer restoring from the snapshot
-If there is a recent snapshot available, you should use snapshot restore
-instead of this tool. Snapshot restore gives you a *consistent* data back
-in time. This tool usage gives you *inconsistent* data back in time.
-
-
-[float]
-=== Command line tool
-`elasticsearch-node` tool has two modes:
-
-* `elastisearch-node unsafe-bootstap` could be used to bootstrap master-eligible
-node in the cluster, where the majority of master eligible nodes is lost, but
- there is at least one master-eligible node available.
-* `elastisearch-node detach-cluster` could be used to detach data nodes from
-the cluster when the all master eligible nodes are lost in the cluster.
+Sometimes {es} nodes are temporarily stopped, perhaps because of the need to
+perform some maintenance activity or perhaps because of a hardware failure.
+Once the temporary condition has been resolved you should restart the node and
+it will rejoin the cluster and continue normally. Depending on your
+configuration, your cluster may be able to remain completely available even
+while one or more of its nodes are stopped.
+
+Sometimes it might not be possible to restart a node after it has stopped. For
+example, the node's host may suffer from a hardware problem that cannot be
+repaired. If the cluster is still available then you can start up
+a fresh node on another host and {es} will bring this node into the cluster in place
+of the failed node.
+
+Each node stores its data in the data directories defined by the
+<<path-settings,`path.data` setting>>. This means that in a disaster you can
+also restart a node by moving its data directories to another host, presuming
+that those data directories can be recovered from the faulty host. Note that it
+is not possible to restore the data directory from a backup because this will
```

Comment:
@DaveCTurner Not sure what you mean by "is not possible to restore the data directory from a backup". 
If you copy full data folder to another node, this node will be indistinguishable from the previous node.

Reply:
Copying the literal data directory of the dead node somewhere else, post-mortem, is ok, but restoring from a backup (which could be who-knows-how stale) is decidedly not. We occasionally see people taking backups of their data directories and causing hassle when they discover that they can't restore from such things, and I saw a risk that this paragraph might be interpreted wrongly by those kinds of people.

Reply:
@DaveCTurner Now I see what you mean, can we re-phrase it like "Note that if you have previously taken a backup of the data folder of the stopped node, you can not restore from it, you need a data folder state at the moment this node was stopped"

Reply:
The trouble with the phrase "if you have previously taken a backup of the data folder" is that it suggests this is a thing you might try and do, and I think we should avoid making that suggestion. Technically you _can't_ take a backup of the data folder: a backup from which you cannot ever safely restore isn't really a backup at all 🤔.

Reply:
@DaveCTurner I agree. In this case, probably it makes sense to remove this sentence at all because it's confusing?
Also, the better place for explaining that there is no reason to copy data folders because it's not possible to restore from them is on the snapshot page.